data/strategies/StrategyAH2EarncyDAI.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "strategy",
+  "name": "StrategyAH2EarncyDAI",
+  "description": "Lends DAI on Alpha Homora v2 to generate interest. Users of Alpha Homora borrow DAI to perform leveraged yield-farming on Alpha’s platform.",
```

Comment:
I haven't seen any reference to outside users in other descriptions. It might confuse some people